src/com/nicoles_change_maker/CoinMap.java

```diff
@@ -0,0 +1,23 @@
+package com.nicoles_change_maker;
+import java.util.Iterator;
+import java.util.LinkedHashMap;
+/**
+ * Created by bears8yourface on 1/5/16.
+ */
+public class CoinMap {
+
+    public static final LinkedHashMap dictionary = new LinkedHashMap<String, Double>() {
```

Comment:
I mentioned this before, but you should more or less never expose a public instance variable. You have a bit more protection here because it's `final`, but it's always best to expose behavior, not data.